assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/versions/details-versions/details-versions-container.tsx

```diff
@@ -58,25 +58,46 @@ export const DetailsVersionsContainer = ({
     Promise.all(versionPromises)
       .then((responses): void => {
         const tempVersionData: any[] = []
-        const data = responses[0].data
+        const dataRaw = responses[0].data as ImageVersion
+        const metadata = dataRaw.metadata
+
+        const data: Partial<ImageVersion> = { ...dataRaw }
+        delete data.metadata
+
         for (const key in data) {
           tempVersionData.push({
             [i18n.t('field')]: i18n.t(`version.${key}`)
           })
         }
 
+        for (const meta of metadata) {
+          data[`${meta.name} (${meta.type})`] = meta.data
+          tempVersionData.push({
+            [i18n.t('field')]: `${meta.name} (${meta.type})`
```

Comment:
We should use the `t`-function from `useTranslation` instead.

The normal i18n module has no reactivity inbuilt, this could cause problems in the future when the module is loaded before the translations are present.